RayLin.md

```diff
@@ -1,22 +1,12 @@
-# 07/11 ~ 07/15
+# 07/18 ~ 07/22
 
 - [Video Control]
-  - Bug 1281414 - [webvtt] Update the design of closed caption button icon
-    - CC'd Carol, attcah comparison screenshot and reiterate intention
-  - Bug 883122 - [webvtt] Implement the rules for constructing a chapter tree from a text track
-    - build test page and study algorithm
-  - Bug 1281414 - [webvtt] Update the design of closed caption button icon
-    - discuss with Peko
-      - refresh primitive features without regard to Bug 883122
-  - Bug 681553 - [meta] Make HTML5 <audio> controls less of a hack
-    - NI :Dolske
-    - ask if it doable and what's the expected result
-  - Bug 855149 - Make <input type=range> usable for media controls
-    - need to figure out event issues:
-      - how to avoid event propagating to outside
-      - how to implement system group event option without XBL
-
-- [Firefox for Android]
-  - Bug 1282678 - [webvtt] Consider adding closed caption button to mobile video control
-    - :barbars is ok with it, and added :antlam for feedback
-
+  - Bug 1271765 - Visual refresh of media controls
+    - UI discussion with Peko
```

Comment:
It would be wonderful if there can be a summery of the discussion documented on Bugzilla so your future reviewer can see it!